packages/toolkit/src/test/srcShared/fs.test.ts

```diff
@@ -6,13 +6,14 @@
 import assert from 'assert'
 import * as vscode from 'vscode'
 import * as path from 'path'
-import { existsSync, mkdirSync, promises as fsPromises, readFileSync, rmSync, writeFileSync } from 'fs'
+import { existsSync, mkdirSync, promises as fsPromises, readFileSync, rmSync } from 'fs'
 import { FakeExtensionContext } from '../fakeExtensionContext'
 import { fsCommon } from '../../srcShared/fs'
 import * as os from 'os'
 import { isMinimumVersion } from '../../shared/vscode/env'
 import Sinon from 'sinon'
 import * as extensionUtilities from '../../shared/extensionUtilities'
+import { outputFile } from 'fs-extra'
```

Comment:
can avoid this. we have `toFile` already. https://github.com/aws/aws-toolkit-vscode/blob/66cc926ab7bae2397f64396bb59720f6f38f877d/packages/toolkit/src/test/testUtil.ts#L26

Reply:
Thanks for pointing out. It will require a good amount of changes to add a `mode` option.
But I've updated `makeFile()` to now use this function + `chmod` and gotten rid of the `fs-extra` import.